components/connector/internal/tokens/service.go

```diff
@@ -34,6 +37,7 @@ func (svc *tokenService) CreateToken(clientId string, tokenType TokenType) (stri
 		ClientId: clientId,
 	}
 
+	log.Debugf("Storing token for %s with id %s in the cache", tokenType, clientId)
```

Comment:
It would be useful if you could log for how long the token is cached (Caching one-time token for X mins